default.json

```diff
@@ -145,19 +144,17 @@
       "matchPackageNames": ["braid-design-system", "sku", "skuba"],
       "matchPackagePatterns": ["^@?seek", "seek$", "^@vanilla-extract/"],
 
-      "prCreation": "immediate",
```

Comment:
This kinda sucks 😞

Reply:
Am I wrong that if the outside is `prCreation: "immediate"` then that will apply to all of them? Is that how it works?